app/request_handlers.py

```diff
@@ -617,15 +571,11 @@ async def post(self, request):
         if not form_valid:
             logger.info('form submission error',
                         client_ip=request['client_ip'])
-            return {
-                'form_value_name_first_name': data.get('name_first_name'),
-                'form_value_name_last_name': data.get('name_last_name'),
```

Comment:
If errored 'First Name' or 'Last Name' are not passed back into the form anymore so for instance you get "Enter your last name" but your first name hasn't been saved if you entered it?